web-local/start-zookeeper.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
```

Comment:
We should make this file a separate pool request.

Reply:
@bdevore17 file has been removed, please check again